applications/tari_dan_wallet_daemon/src/handlers/accounts.rs

```diff
@@ -902,19 +904,21 @@ pub async fn handle_transfer(
             method: "deposit".to_string(),
             args: args![Workspace("bucket")],
         },
-        Instruction::CallMethod {
-            component_address: source_account_address,
-            method: "pay_fee".to_string(),
-            args: args![max_fee],
-        },
     ]);
 
+    fee_instructions.append(&mut vec![Instruction::CallMethod {
```

Comment:
Why this change?

Suggest adding the instruction to the end and using `Vec::extend` rather than `Vec::append`.
```rust
// vec![....] allocates on the heap, then append takes each allocated item and the vec is dropped
    fee_instructions.append(&mut vec![Instruction::CallMethod {
```

`Vec::entend`  takes any iterator, in this case, a static array of instructions 
```suggestion
    fee_instructions.extend([Instruction::CallMethod {
```